pom.xml

```diff
@@ -38,6 +38,7 @@
     <module>document</module>
     <module>security</module>
     <module>migration</module>
+    <module>optimize</module>
```

Comment:
Note: adding the module here does not mean including optimize in the jar. This has the effect of building optimize together with the rest of the project, so that the artifact is built from the code instead of being downloaded from camunda-nexus, with potential cache poisoning effects